src/components/Auth/Register.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react"
```

Comment:
The UI in this component and in the Signin component is not done. This is just so that the functionality is implemented. In this screen, the confirm form should only be visible after the user clicks register. This is something that can be tackled in a UI follow-up